setup.py

```diff
@@ -47,19 +47,25 @@
 """
 
 
-setup(name='ntplib',
-      version='0.3.4',
-      description='Python NTP library',
-      author='Charles-Francois Natali',
-      author_email='cf.natali@gmail.com',
-      url='https://github.com/cf-natali/ntplib',
-      py_modules=['ntplib'],
-      license='MIT',
-      classifiers=[
-          'License :: OSI Approved :: MIT License',
-          'Programming Language :: Python',
-          'Operating System :: OS Independent',
-          'Topic :: System :: Networking :: Time Synchronization'
-      ],
-      long_description=long_description
-     )
+setup(
+    name="ntplib",
+    version="0.3.5",
+    description="Python NTP library",
+    author="Charles-Francois Natali",
+    author_email="cf.natali@gmail.com",
+    url="https://github.com/cf-natali/ntplib",
+    py_modules=["ntplib"],
+    license="MIT",
+    classifiers=[
+        "License :: OSI Approved :: MIT License",
+        "Programming Language :: Python",
+        "Operating System :: OS Independent",
+        "Topic :: System :: Networking :: Time Synchronization",
+        "Programming Language :: Python :: 2.7",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
Is it really necessary to list all the supported Python versions?
I don't really want to have to update this whenever a new Python version is released.

Reply:
its not necessary but its useful

eg you could have this banner in the readme file as a result
![Screenshot 2021-03-02 at 08 37 17](https://user-images.githubusercontent.com/2742870/109610656-b67c1380-7b35-11eb-88d3-da59e3e9f2fb.png)

Reply:
> I don't really want to have to update this whenever a new Python version is released.
new major python version is released once a year, the other releases are just bugfix releases which occur every +/- 3 months. these classifiers are for tracking major python versions only not every "bugfix" release and it is okay if these are not updated immediately.